.travis.yml

```diff
@@ -1,21 +1,32 @@
+sudo: false
 language: scala
 
 jdk:
   - oraclejdk8
 
+scala:
+  - 2.10.6
+  - 2.12.0-M5
+  - 2.12.0-RC1
+  - 2.12.0-RC2
+
+matrix:
+  include:
+    - scala: 2.12.0-M4
+      env: TEST_SCRIPTED=1
+      jdk: oraclejdk8
+    - scala: 2.11.8
+      env: TEST_SCRIPTED=1
+      jdk: oraclejdk8
+
```

Comment:
So does the matrix thing add two additional builds with `TEST_SCRIPTED=1` defined? I haven't used it before.